backend/fms_core/serializers.py

```diff
@@ -381,6 +382,8 @@ def serialize_sample_export(sample: Sample) -> Dict[str, Any]:
         'quality_flag': None if sample.quality_flag is None else ("Passed" if sample.quality_flag else "Failed"),
         'quantity_flag': None if sample.quantity_flag is None else ("Passed" if sample.quantity_flag else "Failed"),
         'projects': ''.join([project.name for project in sample.projects.all()]) if sample.projects.all() else None,
+        'projects_external_id': ''.join([project.external_id for project in sample.projects.all()]) if sample.projects.all() else None,
```

Comment:
Not sure we need to export them. They should be in the project endpoint but i would not add them to samples or samples/list_export